### PR TITLE
WEBUI-569: lint and test nuxeo-elements ondemand github workflow

### PR DESCRIPTION
**Pull request by richardsd**
Improvements:

- Installs, Lints and tests elements
- Similar strategy used in old Jenkinsfile for installing `nuxeo-elements` in WebUI before running the functional tests

### CODE REVIEW

File: .github/workflows/ondemand-main.yaml [hunk 1]
```diff
@@ -8,33 +8,34 @@ on:
         default: 'maintenance-3.0.x'
         required: true
       sauce_labs:
-        description: 'Should unit tests be run on Sauce Labs?'
+        description: 'Run Unit tests on Sauce Labs?'
```
Comment (Gabez0r) [inline .github/workflows/ondemand-main.yaml]: For consistency with the descriptions below:
```suggestion
        description: 'Run unit tests on Sauce Labs?'
```

File: .github/workflows/ondemand-main.yaml [hunk 2]
```diff
@@ -77,26 +90,68 @@ jobs:
...
+
+      - name: Elements Unit tests (Sauce Labs)
+        if: ${{ github.event.inputs.skip_unit_tests == 'false' && github.event.inputs.sauce_labs == 'true' }}
+        env:
+          SAUCE_USERNAME: nuxeo-web-ui
+          SAUCE_ACCESS_KEY: ${{ secrets.SAUCE_ACCESS_KEY }}
+        run: |
+          pushd nuxeo-elements
+          npm run test
+          popd
```
Comment (Gabez0r) [inline .github/workflows/ondemand-main.yaml]: Except for the name and the if condition, aren't these two blocks doing exactly the same? 🤔

Comment (richardsd) [reply .github/workflows/ondemand-main.yaml]: No, you're right. The first one shouldn't have the Sauce Labs environment variables. Thanks, copy&paste 😬

**Comment by richardsd** [comment]
Screenshot 2021-11-17 at 12 51 08

**Review by Gabez0r — DISMISSED** [review]

**Review by Gabez0r — APPROVED** [review]

**Review by mnixo — APPROVED** [review]

**Pull request merged**